lib/utils/styles.dart

```diff
@@ -39,7 +39,11 @@ abstract class Styles {
   }
 
   static TextStyle articleListTitleStyle() {
-    return TextStyle(fontSize: 15, fontWeight: FontWeight.bold, color: Color(black), decorationColor: Color(primaryGreen));
+    return TextStyle(fontSize: 17, fontWeight: FontWeight.w700, color: Color(darkBlue));
```

Comment:
I see that styles are now created per widget, are these styles (and associated global colors) still in use?

Reply:
Global colors are now unused, when we'll migrate all the files into new coding style, this global colors are going to be deleted, or refactor to have only the basic ones